Github-Org-Management-Policy.md

```diff
@@ -0,0 +1,50 @@
+# Node.js GitHub Organization Management Policy
+
+The Node.js Foundation Github Organization (https://github.com/nodejs) is
+provided as a development resource by the Node.js Foundation under the
+operational direction of the Node.js Technical Steering Committee (TSC).
+
+## Organization Roles
+
+Individual users within a GitHub organization may be granted one of three
+distinct roles: Member, Owner, or Billing manager. The permissions granted
+to each role are documented in the GitHub documentated [Permission levels for
+an organization](https://help.github.com/articles/permission-levels-for-an-organization/).
+
+### Billing manager
+
+All Billing managers for the Node.js Foundation GitHub Organization shall
+be determined by the Node.js Foundation Board of Directors with notification
+given to the TSC.
```

Comment:
Our org is free, comp'd because we are a non-profit (and because we sort of helped out GitHub in beta testing their new orgs a year or so back).

This email address may still be used if you're trying to recover an org someone has taken over but that may be an outdated notion, possibly before the new orgs system, because any Owner can swap out that value now so it wouldn't make a very good recovery attribute.